### DISCUSSION TITLE
How to remove notification center icon with Windows 10 taskbar?

### DISCUSSION THREAD

**Discussion by ahaas25** [valinet/ExplorerPatcher#354] [Q&A] [answered]
https://i2.wp.com/techdirectarchive.com/wp-content/uploads/2020/03/screenshot-2020-03-14-at-10.13.38.png?resize=595%2C447&ssl=1

This is the icon in question

**Comment by valinet** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/6503598/141704990-c4457ecb-ed87-4297-9587-4e88dba39da5.png)

![image](https://user-images.githubusercontent.com/6503598/141705027-126ec1fe-db31-4e72-bbf4-6a4fbd7bc87b.png)

![image](https://user-images.githubusercontent.com/6503598/141705068-dda02b5b-df76-4d12-8b70-8ab7983c3d07.png)

**Reply by ahaas25** [reply]
![image](https://user-images.githubusercontent.com/94150901/141705157-78f23378-649d-4975-8b1f-55d592dfc62b.png)

This is what I see when I right-click on my taskbar. Is there another way to access that menu?

**Reply by valinet** [reply]
Right click on the clock.

Alternatively:

![image](https://user-images.githubusercontent.com/6503598/141705526-efce8d88-1d1e-4ad2-a092-d5c96d6a2ccc.png)

![image](https://user-images.githubusercontent.com/6503598/141705551-cb9c9ec1-b074-43fd-84a2-eab20e461f9f.png)

![image](https://user-images.githubusercontent.com/6503598/141705568-2f15e4bb-2802-4d70-bc03-dd10405a0e95.png)

**Comment by ahaas25** [comment]
Thank you!!!

**Comment by AndyIreland** [comment]
Thank you @valinet  you directed me from : https://github.com/valinet/ExplorerPatcher/issues/355 

But this is for windows 10 and I have windows 11 installed and trying out ExplorerPatcher-master to git my old look windows 10 taskbar back , but now missing resizing of taskbar buttons and missing clock and date with this patch in the system tray.

If I right click the taskbar to change properties I am getting the windows 11 properties for changing the properties and not the windows 10 properties for changing the taskbar properties. 

This is the properties I am getting. The windows 11 properties so no way of changing the taskbar button sizes (make them small icons) and no facility to add clock and date to system  tray. 

![windows11 taskbar properties](https://user-images.githubusercontent.com/49172217/141788673-14d39f59-d30d-41f1-8c1d-95c923a0eb56.jpg)

and here is other window :
![taskbar properties](https://user-images.githubusercontent.com/49172217/141788939-9c83ef3a-8b40-46da-bedc-2b9a7b84ebaa.jpg)

Do you reckon I have not installed ExplorerPatcher properly on my windows 11?

Thanks.

**Reply by valinet** [reply]
Try running the Properties GUI manually: `rundll32 C:\Windows\dxgi.dll,ZZGUI`.

**Reply by AndyIreland** [reply]
> Try running the Properties GUI manually: `rundll32 C:\Windows\dxgi.dll,ZZGUI`.

thank you I will give that a go
